tamr_client/operation.py

```diff
@@ -0,0 +1,195 @@
+"""
+See https://docs.tamr.com/new/reference/the-operation-object
+"""
+from copy import deepcopy
+from dataclasses import dataclass
+from time import sleep, time as now
+from typing import Dict, Optional
+
+import requests
+
+from tamr_client import response
+from tamr_client.instance import Instance
+from tamr_client.session import Session
+from tamr_client.types import JsonDict
+from tamr_client.url import URL
+
+
+class NotFound(Exception):
+    """Raised when referencing an operation that does not exist on the server.
+    """
+
+    pass
+
+
+@dataclass(frozen=True)
+class Operation:
+    """A Tamr operation
+
+    See https://docs.tamr.com/new/reference/the-operation-object
+
+    Args:
+        url
+    """
+
+    url: URL
+    id: str
```

Comment:
I don't think we should keep track of `id` separately from `url` since that can duplicate information and potentially become inconsistent